src/Orchard.Cms.Web/Startup.cs

```diff
@@ -40,9 +38,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseStaticFiles();
             loggerFactory.AddConsole(Configuration);
```

Comment:
Only thing still to do is to remove `loggerFactory.AddConsole(Configuration); ` line and adding a console target to NLog config file. https://github.com/NLog/NLog/wiki/Console-target